### PR TITLE
[JA] Add 'Code of conduct for modding and mapping'

### PR DESCRIPTION
**Pull request by baar-vendel**
Proposal for the Japanese translation of the wiki page `Code of conduct for modding and mapping`, asked by VINXIS in August

### CODE REVIEW

**Comment by TicClick** [comment]
@peppy could you please proofread this before merge if you have time?

**Review by peppy — CHANGES_REQUESTED** [review]

**Comment by peppy** [comment]
I had a pending review I forgot to submit here, sorry!

Mostly just terminology standardisation. Reads very well otherwise. I've made recommendations to use english terms in line with how we write them on other wiki pages and crowdin.

File: wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/ja.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Code of conduct for modding and mapping
+
+**Code of conduct for modding and mapping**とは、osu!の[マッピング](/wiki/Beatmapping)と[モッディング](/wiki/Modding)のエコシステム全体に適用されるルールとガイドラインからなる行動規範のことで、[コミュニティルール](/wiki/Rules)と[ランキング基準](/wiki/Ranking_Criteria)の延長線上にあります。スレッドでのマッピングとモッディングの議論が、建設的かつ前向きで生産的な環境で行われるようにするため、全員が同じ規範を保つ必要があります。モッディングとマッピングのエコシステムに参加する以上は、当規範に従わなければなりません。これらの規則を破ると[アカウントにペナルティが課される](/wiki/Silence)可能性があります。
+
+## 振る舞いと行動
+
+- Mapperが望む場合、Moddingの目的は[Ranked](/wiki/Beatmap/Category#ranked)が可能になるように譜面を改善する手助けをすることです。もし協力する意志がないのであれば、邪魔せず議論に関わらないようにしましょう。
+- Moddingをするときは、誰に、どのような目的で、何をして欲しいか(どのような反応を期待しているのか)を明確にしましょう。そうすることでこちらの意図が確実に伝わり、理解しやすい投稿内容になります。
+  - 新規MapperやModderを敬遠しないでください。彼らはインプットから最も恩恵を受けられます。忍耐強く耳を傾け、はやく成長できるようにできるだけ多くの説明をしてあげましょう。
+  - もしあなたが新規Modderであれば、恥ずかしがらずそう公言しましょう。コミュニティーの経験豊富なメンバーが正しい方向に導いてくれます。誰もがかつては新人で、あなたと同じ立場でした。
```
Comment (baar-vendel) [inline wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/ja.md:10]:
```suggestion
  - もしあなたが新規Modderであれば、恥ずかしがらずそう公言しましょう。コミュニティの経験豊富なメンバーが正しい方向に導いてくれます。誰もがかつては新人で、あなたと同じ立場でした。
```

File: wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/ja.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Code of conduct for modding and mapping
```
Comment (baar-vendel) [inline wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/ja.md:1]:
```suggestion
# ModdingとMappingにおける行動規範
```

File: wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/ja.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
+# ModdingとMappingにおける行動規範
+
+**Code of conduct for modding and mapping**とは、osu!の[Mapping](/wiki/Beatmapping)と[Modding](/wiki/Modding)のエコシステム全体に適用されるルールとガイドラインからなる行動規範のことで、[コミュニティルール](/wiki/Rules)と[ランキング基準](/wiki/Ranking_Criteria)の延長線上にあります。スレッドでのMappingとModdingの議論が、建設的かつ前向きで生産的な環境で行われるようにするため、全員が同じ規範を保つ必要があります。ModdingとMappingのエコシステムに参加する以上は、当規範に従わなければなりません。これらの規則を破ると[アカウントにペナルティが課される](/wiki/Silence)可能性があります。
```
Comment (baar-vendel) [inline wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/ja.md:3]:
```suggestion
**ModdingとMappingにおける行動規範**とは、osu!の[Mapping](/wiki/Beatmapping)と[Modding](/wiki/Modding)のエコシステム全体に適用されるルールとガイドラインからなる行動規範のことで、[コミュニティルール](/wiki/Rules)と[ランキング基準](/wiki/Ranking_Criteria)の延長線上にあります。スレッドでのMappingとModdingの議論が、建設的かつ前向きで生産的な環境で行われるようにするため、全員が同じ規範を保つ必要があります。ModdingとMappingのエコシステムに参加する以上は、当規範に従わなければなりません。これらの規則を破ると[アカウントにペナルティが課される](/wiki/Silence)可能性があります。
```

**Comment by baar-vendel** [comment]
Thank you for the review! I changed everything you said, and also translated the title.

**Comment by baar-vendel** [comment]
@TicClick Fixed the issues above. What should I do next?

**Comment by TicClick** [comment]
@baar-vendel I planned to merge it after 2+ weeks w/o review, but now that it's been reviewed I guess peppy will take over and merge it himself

**Review by peppy — APPROVED** [review]

**Pull request merged**